src/middleware/validation.schema.js

```diff
@@ -136,6 +136,32 @@ const updateArticleSchema = {
   }
 };
 
+const publishedArticle = {
```

Comment:
Add a validation for published date to make sure it's a date string, but don't mark it as required